### PR TITLE
fix: allow space in cafe css project step 32

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Ref #52549

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866d28d7ad0de6470505.md [hunk 1]
```diff
@@ -16,19 +16,22 @@ Add the class name `flavor` to the `French Vanilla` `p` element.
...
+const elements = document.querySelectorAll('.flavor');
+assert.lengthOf(elements, 1);
 ```
 
 Your `flavor` class should be on the `p` element with the text `French Vanilla`.
 
 ```js
-assert($('.flavor')[0].innerText.match(/French Vanilla/i));
+const el = document.querySelector(".flavor");
+assert.match(el.innerText,/French Vanilla/i);
```
Comment (lasjorg) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866d28d7ad0de6470505.md:34]: Shouldn't this be an equal check for the string value and not a case-insensitive regex match?

Comment (a2937) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866d28d7ad0de6470505.md:34]: I don't think it really matters that much here. As long as the text is close enough, it's probably fine.

Comment (a2937) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866d28d7ad0de6470505.md]: For the sake of clarity, I am going to apply this suggestion.

**Review by huyenltnguyen — APPROVED** [review]

**Review by lasjorg — APPROVED** [review]
LGTM

**Pull request merged**